Eastern-Research-Group/mywaterway
# Feature/40 educational materials

## Related Issues:
* [https://jira.epa.gov/browse/HMW-40](https://jira.epa.gov/browse/HMW-40)

## Main Changes:
* Added a new page that features educational materials.
* Included a link to the page in the banner icons (the graduation cap).
* Moved content management to the `LookupFilesContext` so it can be served from the S3 Bucket.

## Steps To Test:
1. Click the "Educators" navigation link
2. Check that the links are visible, in this case the "middleschool lesson plan" link
